### ISSUE TITLE
[BUG] Internal Media Player won't start, crashes RaceControl

### ISSUE THREAD

**Issue by Ainson** [robvdpol/RaceControl#231]
Clicking Watch will launch a tiny black window, freezing and finally crashing RaceControl (A message box comes up that RaceControl has stopped working). Everything else works perfectly.

I can't find anything relevant in the log.

I've disabled Firewall and Anti-virus without help.

https://i.imgur.com/uONjgnX.jpg

Windows 7 x64
RaceControl 2.4.0

- Subscription type Pro
- Using VPN
- Latest .NET 

... Is the internal player inside the RaceControl.exe or what?

Any ideas?

**Comment by SuRGeoNix** [comment]
I'm afraid that an issue with Flyleaf internal player that it doesn't fully support Windows 7. It would be possible to work by disabling the video acceleration (but currently RaceControl does not expose Flyleaf's confiiguration). You can confirm that the issue is this by downloading [flyleaf](https://github.com/SuRGeoNix/Flyleaf/releases/latest) and trying to disable the video acceleartion(Right click -> Video -> HW Accelearation)

**Comment by Ainson** [comment]
I downloaded and tried FlyLeafPlayer, and it plays the F1TV url, but only if hardware acceleration is off.

With hardware acceleration on, it doesn't crash or give any error messages, it just doesn't play.

I tried to copy the FlyLeafPlayer config file Flyleaf.Config.xml to RaceControl folder, but no luck.

(Graphics card is AMD Radeon HD 7700)

That's it then?

**Comment by SuRGeoNix** [comment]
Yes, that's Flyleaf's problem that will not fall back to auto disable the video acceleration. The config file is not supported by RaceControl, its the application responsibly to maintain it. You should consider updating your Win 7, Rob should consider adding support for Flyleaf's config, and I should consider adding full support for Win 7 :)

**Comment by Ainson** [comment]
Ok, no biggie. I just wanted to have sync on several streams and thought maybe the internal player would sort it.

I know there's a sync feature for MPV, but that's not working so good here.

Maybe I'll look to do something manually with Autohotkey instead, or download videos first and use Awesome Video Player.

Thanks.
